README.md

```diff
@@ -1 +1,735 @@
+
+*Aggregated*:
+
+* AGGREGATE_ASK, AGGREGATE_BID - Ask, Bid side of an aggregate order book (futures depth and Nasdaq Level II).
+
+  ---
+ *Full order depth*:
+   
+  * BATE - Cboe EU BXE (BATE).
+  * BYX - Cboe BYX.
+  * BZX - Cboe BZX.
+  * BXTR - Cboe EU SI (Systematic Internaliser).
+  * BI20 - BIST Top20 Orders (Level2+).
+ * CHIX - Cboe EU CXE (Chi-X).
+ * CEUX - Cboe EU DXE.
+  * CFE - Cboe CFE.
+  * C2OX - Cboe C2.
+  * DEA - Cboe EDGA.
+  * DEX - Cboe EDGX.
+  * ERIS - ErisX.
+  * ESPD - Nasdaq E-speed.
+  * FAIR - FairX.
+  * GLBX - Globex.
+  * ICE - ICE Futures US/EU.
+   * IST - Borsa Istanbul.
+   * MEMX - Members Exchange.
+  * NTV - Nasdaq TotalView.
+  * NFX - Nasdaq NFX.
+* SMFE - SmallEx.
+ * XNFI - Nasdaq NFI.
+ * XEUR - Eurex.
+ ---
+*Price level book*:
+ 
+  * glbx - CME Globex.
+  * iex - IEX.
+  * memx - Members Exchange.
+  * ntv - Nasdaq TotalView.
+  * smfe - SmallEx.
+  * xeur - Eurex.
+    
+
+| :information_source: CODE SAMPLE: take a look at `order_sources_ptr` usage in  [PriceLevelBookSample](https://github.com/dxFeed/dxfeed-c-api/blob/master/samples/PriceLevelBookSample/PriceLevelBookSample.c#L334)|
+| --- |
+
+|:white_check_mark: READ MORE: [Order sources](https://kb.dxfeed.com/en/data-model/qd-model-of-market-events.html#UUID-858ebdb1-0127-8577-162a-860e97bfe408_para-idm53255963764388)|
+| --- |
+   
+---
+
+## Usage
+
+#### Create connection
+
+```csharp
+//connection handler declaration
+dxf_connection_t connection;
+//creating connection result declaration
+ERRORCODE connection_result;
+//creating connection
+connection_result = dxf_create_connection(dxfeed_host, on_reader_thread_terminate, NULL, NULL, NULL, NULL, &connection);
+```
+
+---
+
+#### Create subscription
+
+```csharp
+//subscription handler declaration
+dxf_subscription_t subscription;
+//creating subscription result declaration
+ERRORCODE subscription_result;
+//creating subscription handler
+subscription_result = dxf_create_subscription(connection, event_type, &subscription);
+```
+
+---
+
+#### Setting up contract type
+
+```csharp
+//creating subscription handler with Ticker contract
+subscription_result = dxf_create_subscription_with_flags(connection, event_type, dx_esf_force_ticker, &subscription);
+```
+
+---
+
+#### Setting up symbol
+
+```csharp
+//adding array of symbols
+dxf_add_symbols(subscription, (dxf_const_string_t*)symbols, symbol_count)
+```
+
+---
+
+#### Setting up Order source
+
+```csharp
+//setting Nasdaq TotalView FOD source
+dxf_set_order_source(subscription, "NTV")
+```
+
+---
+
+#### Quote subscription
+
+```cpp
```

Comment:
We can use a simpler example.  
https://www.dxfeed.com/api/c-and-net-apis/ :
```cpp
#include <stdio.h>
#include <time.h>
#include "DXFeed.h"
void print_timestamp(dxf_long_t timestamp) {
	wchar_t timefmt[80];
	struct tm* timeinfo;
	time_t tmpint = (time_t)(timestamp / 1000);
	timeinfo = localtime(&tmpint);
	wcsftime(timefmt, 80, L"%Y%m%d-%H%M%S", timeinfo);
	wprintf(L"%ls", timefmt);
}
void listener(int event_type, dxf_const_string_t symbol_name, const dxf_event_data_t* data, int data_count,
			  void* user_data) {
	wprintf(L"%Quote{symbol=%ls, ", symbol_name);
	dxf_quote_t* q = (dxf_quote_t*)data;
	wprintf(L"bidTime=");
	print_timestamp(q->bid_time);
	wprintf(L" bidExchangeCode=%c, bidPrice=%f, bidSize=%i, ", q->bid_exchange_code, q->bid_price, q->bid_size);
	wprintf(L"askTime=");
	print_timestamp(q->ask_time);
	wprintf(L" askExchangeCode=%c, askPrice=%f, askSize=%i, scope=%d}\n", q->ask_exchange_code, q->ask_price,
			q->ask_size, (int)q->scope);
}
int main(int argc, char* argv[]) {
	dxf_connection_t con;
	dxf_subscription_t sub;
	dxf_create_connection("demo.dxfeed.com:7300", NULL, NULL, NULL, NULL, NULL, &con);
	dxf_create_subscription(con, DXF_ET_QUOTE, &sub);
	dxf_const_string_t symbols2[] = {L"IBM", L"GOOG", L"AAPL", L"SPY"};
	dxf_attach_event_listener(sub, listener, NULL);
	dxf_add_symbols(sub, symbols2, sizeof(symbols2) / sizeof(symbols2[0]));
	wprintf(L"Press enter to stop");
	getchar();
	dxf_close_subscription(sub);
	dxf_close_connection(con);
	return 0;
}
```

or 

https://github.com/ttldtor/dxfeed-c-api-sample/blob/default/src/main.cpp